doc/man/zcashd.1

```diff
@@ -264,6 +264,11 @@ Spend unconfirmed change when sending transactions (default: 1)
 If paytxfee is not set, include enough fee so transactions begin
 confirmation on average within n blocks (default: 2)
 .HP
+\fB\-txexpirydelta\fR
+.IP
+Set the number of blocks after which a transaction that has not been
```

Comment:
Non-blocker: i think this doesn't explain what the delta is, that is, its based on the block height at time of transaction creation

Reply:
I'm happy to fix this during the RC (along with other documentation-related changes like #2957).